EDU-16086: Redirects limitations (#2187)
vtexdocs/dev-portal-content: docs/faststore/docs/routing/managing-urls-with-redirects-and-rewrite-paths.mdx

---
title: "Managing URLs with redirects and rewrite paths"
excerpt: "Learn how to configure redirects and rewrites in your FastStore project to manage user access to your content."
---

> In this guide, you will learn how to configure redirects and rewrites in your FastStore project. This enables you to effectively manage user access to your content by automatically directing users from one URL to another or mapping custom URLs to existing page templates.

Reviewer: ```suggestion
In this guide, you will learn how to configure redirects and rewrites in your FastStore project. These tools help you manage user access to your content by automatically directing users from one URL to another or mapping custom URLs to existing page templates.
```

These two tools enhance content organization, help maintain a good user experience, and ensure search engines continue to index the correct content.

> ## Redirect limitations

Reviewer: ```suggestion
## Redirect behavior and caveats
```

> Before configuring redirects, it's important to understand how the redirect feature behaves in FastStore and its limitations.

Reviewer: ```suggestion
Before configuring redirects, it's important to understand how they behave in FastStore and what caveats apply.
```

### Native routes take precedence

> FastStore prioritizes native pages (e.g., product, collection, and brand pages) over redirect rules. If a redirect's source path matches a native page, the redirect will be ignored. For example, if `/fall-sale` is a live native collection page, a redirect rule from `/fall-sale` to `/fall` will not work. The collection page will render instead. To make the redirect work, you must unpublish or disable the native page (`/fall-sale`).

Reviewer: > To make the redirect work, you must unpublish or disable the native page (`/fall-sale`).

Como? Ajustar no Admin/Catalogo ou no CMS? Excluir a página do código fonte? Tem doc sobre isso pra add link?
